src/gui/tray/Window.qml

```diff
@@ -531,27 +533,8 @@ Window {
                                 text: appName
                                 font.pixelSize: Style.topLinePixelSize
                                 icon.source: appIconUrl
-                                width: contentItem.implicitWidth + leftPadding + rightPadding
                                 onTriggered: UserAppsModel.openAppUrl(appUrl)
                                 hoverEnabled: true
-
-                                background: Item {
-                                    width: appsMenu.width
-                                    height: parent.height
-
-                                    Rectangle {
-                                        anchors.fill: parent
-                                        anchors.margins: 1
-                                        color: appEntry.hovered ? Style.lightHover : "transparent"
-                                    }
-                                    
-                                    Accessible.role: Accessible.PopupMenu
-                                    Accessible.name: qsTr("Apps menu")
-                                }
-
-                                Accessible.role: Accessible.MenuItem

```

Comment:
why do you remove the accessibility attached properties ?
I am not sure they are forbidden here ?

Reply:
That was a mistake. Thanks.